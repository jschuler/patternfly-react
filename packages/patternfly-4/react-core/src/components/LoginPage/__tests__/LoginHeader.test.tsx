// Auto-generated do not edit

import * as React from 'react';
import { shallow } from 'enzyme';
import { LoginHeader } from '../LoginHeader';
// any missing imports can usually be resolved by adding them here
import {  } from '..';

describe('LoginHeader test', () => {
  it('LoginHeader should match snapshot', () => {
    const view = shallow(
      <LoginHeader
        children={<>ReactNode</>}
				className={"''"}
				headerBrand={null}
      />);
    expect(view).toMatchSnapshot();
  });
});
