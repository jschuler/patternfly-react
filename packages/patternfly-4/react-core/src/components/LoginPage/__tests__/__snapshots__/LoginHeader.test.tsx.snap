// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginHeader test LoginHeader should match snapshot 1`] = `
<header
  className="pf-c-login__header ''"
>
  ReactNode
</header>
`;
